frontend/src/metabase/public/components/PublicError.jsx

```diff
@@ -16,12 +16,7 @@ const mapStateToProps = (state, props) => ({
 const PublicError = ({ message = t`An error occurred` }) => (
   <EmbedFrame className={CS.spread}>
     <div className={cx(CS.flex, CS.layoutCentered, CS.flexFull, CS.flexColumn)}>
-      <div
-        className={cx(
-          QueryBuilderS.QueryErrorImage,
-          QueryBuilderS.QueryErrorImageNoRows,
-        )}
-      />
+      <NoDataError mb="1rem" />
```

Comment:
`margin-bottom` is from https://github.com/metabase/metabase/blob/f7a7df8d5c49ea37091b20c3868d8c17560a131d/frontend/src/metabase/css/query_builder.module.css#L149-L152

To see this component, visit public dashboard or public question and enter a wrong UUID

![image](https://github.com/metabase/metabase/assets/1937582/9455768d-825d-4673-94ad-c5bd4fc6e8fb)